orchagent/orch.cpp

```diff
@@ -123,6 +120,47 @@ void Consumer::execute()
             m_toSync[key] = KeyOpFieldsValuesTuple(key, op, existing_values);
         }
     }
+}
+
+// TODO: Table should be const
+void Consumer::refillToSync(Table* table)
```

Comment:
It looks to me if we remove "bool Orch::addExistingData(Table *table)" and use bool Orch::addExistingData(const string& tableName) only,  
void Consumer::refillToSync(Table* table) is not needed and quite a few line of redundant code function may be eliminated.

Reply:
```
void Consumer::refillToSync(Table* table)
```
is used for optimization if we have a table object already.

---
In reply to: [207438102](https://github.com/Azure/sonic-swss/pull/551#discussion_r207438102) [](ancestors = 207438102)